packages/core/utils/src/config.ts

```diff
@@ -75,13 +78,13 @@ const getAbsoluteUrl =
       return url;
     }
 
+    const serverConfig = <ServerConfig>config.server;
     const hostname =
-      config.get('environment') === 'development' &&
-      ['127.0.0.1', '0.0.0.0'].includes(config.get('server.host'))
+      config.environment === 'development' && ['127.0.0.1', '0.0.0.0'].includes(serverConfig.host)
         ? 'localhost'
-        : config.get('server.host');
+        : serverConfig.host;
 
-    return `http://${hostname}:${config.get('server.port')}${url}`;
+    return `http://${hostname}:${serverConfig.port}${url}`;
   };
 
 export const getAbsoluteAdminUrl = getAbsoluteUrl('admin');
```

Comment:
Do we still want to expose those APIs when we're trying to encourage calls to strapi.config.get directly? V5 might be a good opportunity to get rid of them. What do you think?

Reply:
config.get('server') still returns an object so here I wouldn't expect sth else.